srcpkgs/ocs-url/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'ocs-url'
+pkgname=ocs-url
+version=3.1.0
+revision=1
+archs="x86_64 x86_64-musl i686"
```

Comment:
No. Should build for all archs with build style.

Reply:
Building for all arches resulted in failed checks, building on arm failed
> `Project ERROR: Unknown module(s) in QT: qml quick svg`

This error shouldn't happen, but does on arm based arches, thus I packaged only for `x86_64 x86_64-musl i686`

Reply:
Try adding `qt5-declarative-devel` to `makedepends`.

Reply:
Alright, Ill added that remove `archs` and re-ccmmit, if the check fails, ill add `archs` back

Reply:
You can also try to build it locally. `echo XBPS_MAKEJOBS=<your_cores> >> etc/conf; ./xbps-src -a aarch64 pkg ocs-url`

Reply:
archs is no-go here. remove, will be easier to see what's wrong and fix.

Reply:
Understandable, I haven't tried to build locally as my laptop is potato grade, and compiling this from source took alot of power and a long time (2+ hours) for me.

Reply:
I'll wrok on dropping `archs`

Reply:
Managed to do just that! :tada: 
> ```bash
> hostmakedepends="qt5-qmake qt5-svg kdeclarative qt5-svg-devel kdeclarative-devel qt5-declarative-devel"
> makedepends="${hostmakedepends}"
> ```